include/stdio.h

```diff
@@ -16,14 +16,13 @@
 #ifndef _LIBPHOENIX_STDIO_H_
 #define _LIBPHOENIX_STDIO_H_
 
-
 #include <arch.h>
 #include <stddef.h>
 #include <stdarg.h>
 #include <sys/types.h>
 
 #ifndef NOMMU
-#define BUFSIZ SIZE_PAGE
+#define BUFSIZ _PAGE_SIZE
```

Comment:
PAGE_SIZE perhaps?

Reply:
Then we need separate header just for PAGE_SIZE to avoid name space pollution. I think that _PAGE_SIZE is reasonable consensus.